### DISCUSSION TITLE
Module to enable vue syntax.

### DISCUSSION THREAD

**Discussion by 13Groszy** [OrchardCMS/OrchardCore#16396] [Q&A]
Hi there, I'm looking for a bit of support.
Can someone shortly explain or navigate, how to use vue components inside liquid/cshtml files (preferred is liquid)?
Documentation around it is pretty much impossible to find, lot of projects in that area are out-dated and not updated for years (still using vue2).
If that help, I aim to create blog-type website with a lot of various widgets so I need to pass the data to those components somehow.

**Comment by hishamco** [comment]
You can check the following exaples:

1. Cultures list

https://github.com/OrchardCMS/OrchardCore/blob/a259ab40ab2682dc6c45e609e71fb76f60cd5c7e/src/OrchardCore.Modules/OrchardCore.Localization/Views/LocalizationSettings.Edit.cshtml#L20-L68

2. Media Application

https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Media/Views/Admin/MediaApplication.cshtml

**Comment by Piedone** [comment]
Also see: https://github.com/Lombiq/Orchard-Vue.js. This is Vue 3.
